content/en/real_user_monitoring/application_monitoring/flutter/integrated_libraries.md

```diff
@@ -216,7 +216,49 @@ final datadogConfig = DatadogConfiguration(
     );
 ```
 
+## Dio
 
+<div class="alert alert-info">
+For most Dio setups, use Datadog Tracking Http Client instead of the specialized Dio interceptor. Only use the Dio interceptor if you're using a non-standard Dio `HttpClientAdapter` that cannot be tracked by Datadog Tracking Http Client.
```

Comment:
```suggestion
For most Dio setups, use Datadog Tracking Http Client instead of the specialized Dio interceptor. Only use the Dio interceptor if you're using a non-standard Dio <code>HttpClientAdapter</code> that cannot be tracked by Datadog Tracking Http Client.
```